dataprofiler/data_readers/csv_data.py

```diff
@@ -57,9 +59,11 @@ def __init__(self, input_file_path=None, data=None, options=None):
         :type options: dict
         :return: None
         """
-        options = self._check_and_return_options(options)
+        if options is not None:
```

Comment:
I think We can remove this if statement, no?

Reply:
that should fix the bug too

Reply:
Done